### ISSUE TITLE
theme new should include flag for uploading existing directory

### ISSUE THREAD

**Issue by ghost** [Shopify/themekit#873]
Currently, `theme new` will generate a new theme in the directory you specify. There should be a way to simply create a new theme id using the existing theme you're in. I'd like to see a flag that specifies ThemeKit to upload and sync the current working directory, but use the new theme id and name. My use case is simply managing new theme releases by creating a new theme id, then publishing the new theme and restarting the process.

**Comment by natereist** [comment]
I agree completely, I am trying to figure out how to add a theme to additional environments using themekit, my use case is I've started a theme in a development store and want to now add this theme using themekit to another store _(i.e production)_.

**Comment by th0rgall** [comment]
I had the same use case, but in my case I actually could upload my existing theme to a new store. I ran `theme new --env=my_env` from within the root of an existing theme. I had already defined `my_env` with a `store` and `password` key in `config.yml` (but not `theme_id`, because that one was to be created). This did actually upload the existing directory.

But that is not what I expected, since it wasn't documented. See this issue: https://github.com/Shopify/themekit/issues/910
